pySMART/smartctl.py

```diff
@@ -54,10 +73,16 @@ def generic_call(self, params: List[str], pass_options=False) -> Tuple[List[str]
         if not self.smartctl_path:
             raise FileNotFoundError("Command smartctl doesn't exist!")
 
+        popen_list = []
+        if self.sudo and os.name == 'posix':
+            popen_list.append('sudo')
```

Comment:
Instead of "sudo", consider usinging `['sudo','-E']` instead... just to preserve user env

Reply:
From man:

```

     -E, --preserve-env
                 Indicates to the security policy that the user wishes to preserve their existing environment variables.  The security policy may return an error
                 if the user does not have permission to preserve the environment.

```